bundle/src/main/java/com/adobe/acs/commons/fam/package-info.java

```diff
@@ -17,5 +17,5 @@
  * limitations under the License.
  * #L%
  */
-@aQute.bnd.annotation.Version("2.2.0")
+@aQute.bnd.annotation.Version("3.0.0")
```

Comment:
sure, but why do those changes require a major version change? It seems like those changes only pose a backwards compatibility issue for providers (i.e. FAM itself), so the major version change is likely due to missing metadata.